scripts/deployment/phase2-assets/collaterals/deploy_origin_eth_mainnet.ts

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs'
+import hre from 'hardhat'
+import { getChainId } from '../../../../common/blockchain-utils'
+import { baseL2Chains, networkConfig } from '../../../../common/configuration'
+import { bn, fp } from '../../../../common/numbers'
+import { expect } from 'chai'
+import { CollateralStatus } from '../../../../common/constants'
+import {
+  PRICE_FEEDS,
+  ORACLE_ERROR,
+  ORACLE_TIMEOUT,
+} from '../../../../test/plugins/individual-collateral/origin/constants'
+import {
+  getDeploymentFile,
+  getAssetCollDeploymentFilename,
+  IAssetCollDeployments,
+  getDeploymentFilename,
+  fileExists,
+} from '../../common'
+import { priceTimeout } from '../../utils'
+import { OETHCollateral } from '../../../../typechain'
+import { ContractFactory } from 'ethers'
+
+async function main() {
+  // ==== Read Configuration ====
+  const [deployer] = await hre.ethers.getSigners()
+
+  const chainId = await getChainId(hre)
+
+  console.log(`Deploying Origin ETH to network ${hre.network.name} (${chainId})
+    with burner account: ${deployer.address}`)
+
+  if (!networkConfig[chainId]) {
+    throw new Error(`Missing network configuration for ${hre.network.name}`)
+  }
+
+  // Get phase1 deployment
+  const phase1File = getDeploymentFilename(chainId)
+  if (!fileExists(phase1File)) {
+    throw new Error(`${phase1File} doesn't exist yet. Run phase 1`)
+  }
+  // Check previous step completed
+  const assetCollDeploymentFilename = getAssetCollDeploymentFilename(chainId)
+  const assetCollDeployments = <IAssetCollDeployments>getDeploymentFile(assetCollDeploymentFilename)
+
+  const deployedCollateral: string[] = []
+
+  /********  Deploy Origin ETH Collateral - wOETH  **************************/
+
+  // Only for Base
+  if (!baseL2Chains.includes(hre.network.name)) {
+    throw new Error(`Unsupported chainId: ${chainId}`)
+  }
+
+  const OETHCollateralFactory: ContractFactory = await hre.ethers.getContractFactory(
+    'OETHCollateral'
+  )
+
+  const collateral = <OETHCollateral>await OETHCollateralFactory.connect(
+    deployer
+  ).deploy(
+    {
+      priceTimeout: priceTimeout.toString(),
+      chainlinkFeed: PRICE_FEEDS.OETH_ETH, // ETH/OETH
+      oracleError: ORACLE_ERROR.toString(),
+      erc20: networkConfig[chainId].tokens.wOETH,
+      maxTradeVolume: fp('1e6').toString(), // $1m,
+      oracleTimeout: 1, // to avoid impact in calculations
+      targetName: hre.ethers.utils.formatBytes32String('ETH'),
+      defaultThreshold: bn(0).toString(), // depeg checks not used
```

Comment:
Interesting, so soft default peg checks are actually defined in the class (the `pegPrice`) and also tested in the testing suite. If this were deployed, the plugin would default 24h after deployment because the ETH/OETH value in the oracle is not exactly 1e18. 

But this is good, because we _do_ want to soft default checks. So all we need to do here is tweak the deployment script to deploy with a nonzero value. I suggest copying the other LSTs and using `fp('0.02').add(ORACLE_ERROR).toString()`, like is done here: https://github.com/reserve-protocol/protocol/blob/76d2c6ac2046203a99fa00c979bc80ebda72b382/scripts/deployment/phase2-assets/collaterals/deploy_rocket_pool_reth_collateral.ts#L75

Reply:
Changed in this commit: [37e9320](https://github.com/reserve-protocol/protocol/pull/1239/commits/37e9320dc2cdb0de61a2b52e9f4784e5d9ec3a75)